### PR TITLE
Fixed loading theme webpack config

### PR DESCRIPTION
**Pull request by sunel**
readFileSync will read it as text file not as javascript file,

### CODE REVIEW

File: gulp/config/buildWebpack.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ const settings = {
             );
             let childConfig = {};
             try {
-                childConfig = fs.readFileSync(parentConfigPath, 'utf-8');
+                fs.existsSync(parentConfigPath);
```
Comment (krzksz) [inline gulp/config/buildWebpack.js]: Since we are using `require` below then this could be removed.

File: gulp/config/buildWebpack.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ const settings = {
...
+                childConfig =  require(parentConfigPath);
```
Comment (krzksz) [inline gulp/config/buildWebpack.js]: Please remove duplicated space after equals sign.

**Review by krzksz — CHANGES_REQUESTED** [review]
Hey, nice catch and thanks for the contribution! Can you please adjust the code according to my comments?

**Review by sunel — COMMENTED** [review]
Updated the fix based on the comment

**Pull request merged**